### DISCUSSION TITLE
核心文档类是干什么用的，怎么一直打不开？

### DISCUSSION THREAD

**Discussion by Yoka416** [sikouhjw/gdutthesis#29] [Q&A] [answered]
核心文档类.cls是干什么用的，用TeXworks一直无法编译打不开？报错显示在第10行 ： you can't use spacefactor' in vertical mode

**Comment by sikouhjw** [comment]
1. `cls` 不是给你编译的
2. 请阅读 https://github.com/sikouhjw/gdutthesis/wiki/latex_introduction

**Comment by zepinglee** [comment]
建议在 README 中写一下编译方式，模板组成可以略去。

**Comment by sikouhjw** [comment]
> 建议在 README 中写一下编译方式，模板组成可以略去。

在用户手册写了怎么编译，在 README 写了要看用户手册。
反正都是不看……

**Comment by zepinglee** [comment]
不要假定用户会去阅读文档。对于新鸟来说，先编译出示例最重要。

**Comment by sikouhjw** [comment]
> 不要假定用户会去阅读文档。对于新鸟来说，先编译出示例最重要。

问题是连主文件都不知道是哪个，这说明一点基础都没有……救不了

**Comment by Yoka416** [comment]
> 1. `cls` 不是给你编译的
> 2. 请阅读 https://github.com/sikouhjw/gdutthesis/wiki/latex_introduction

谢谢！

**Comment by Yoka416** [comment]
> > 不要假定用户会去阅读文档。对于新鸟来说，先编译出示例最重要。
> 
> 问题是连主文件都不知道是哪个，这说明一点基础都没有……救不了

没有，文档我都看了，也知道在main文件的基础上进行修改，仅仅是不知道.cls文件什么用，可能资料还是看的不够仔细，万分抱歉。

**Comment by sikouhjw** [comment, marked as answer]
> > > 不要假定用户会去阅读文档。对于新鸟来说，先编译出示例最重要。
> > 
> > 
> > 问题是连主文件都不知道是哪个，这说明一点基础都没有……救不了
> 
> 没有，文档我都看了，也知道在main文件的基础上进行修改，仅仅是不知道.cls文件什么用，可能资料还是看的不够仔细，万分抱歉。

cls 是文档类文件，用来定制模板样式、接口，用户一般不需要去管。

**Comment by Yoka416** [comment]
> > > > 不要假定用户会去阅读文档。对于新鸟来说，先编译出示例最重要。
> > > 
> > > 
> > > 问题是连主文件都不知道是哪个，这说明一点基础都没有……救不了
> > 
> > 
> > 没有，文档我都看了，也知道在main文件的基础上进行修改，仅仅是不知道.cls文件什么用，可能资料还是看的不够仔细，万分抱歉。
> 
> cls 是文档类文件，用来定制模板样式、接口，用户一般不需要去管。

嗯嗯好的，明白了，谢谢！
